packages/pymedphys_analysis/src/pymedphys_analysis/gamma/_level1/gammafilter.py

```diff
@@ -33,7 +34,7 @@
 
 import numpy as np
 
-from pymedphys.dicom import coords_from_xyz_axes
+from pymedphys_dicom.dicom import coords_from_xyz_axes
```

Comment:
To keep the subpackages independent of the main package we need to only import other subpackages.

Also, I need to get that layer linter asap, because the moment pymedphys_dicom was used within pymedphys_analysis we had a circular dependency...

I have just refactored things so that the analysis depends on dicom, but prior to this pull request it was the other way round...